### DISCUSSION TITLE
Is it possible to have different data_start_values for different units?

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1453] [Q&A]
Asked by @WohnungNDS as follow up in https://github.com/open-sdg/open-sdg/discussions/1452

Another question: Is it possible to have different data_start_values for different units?
in my case I have numbers (in 1000) and percent 

If percent is selected "Anteil der Personen mit MIgrationshintergrund im weiteren Sinne" is shown.
hitting "in 1000" I have to change th disaggregation manually to e.g. "Personen ohne Migrationshintergrund" before the data is shown.....

[indicator](https://wohnungnds.github.io/IM_Site/1-3-1/)

[metadata](https://github.com/WohnungNDS/IM_Data/edit/develop/meta/1-3-1.md)

**Comment by LucyGwilliamAdmin** [comment]
@WohnungNDS

Yes, it is possible to set start values for a specific unit (or series). You can make use unit/series-specific config syntax - see guidance here: https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#note-about-unit-specific-and-series-specific-settings

For example you should be able to use:

```
data_start_values:
  - unit: in 1000
    field: Migrationshintergrund
    value: Personen ohne Migrationshintergrund
```

**Reply by WohnungNDS** [reply]
Great! I will try it in a couple of days!
